### PR TITLE
Update README.md.

### PR DESCRIPTION
**Pull request by beorn7**
Related to users building from head and getting confused...

@juliusv @fabxc

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,18 +24,23 @@ Prometheus' main distinguishing features as compared to other monitoring systems
...
 There are various ways of installing Prometheus.
 
-### Precompiled packages
+### Precompiled binaries
 
-We plan to provide precompiled binaries for various platforms and even
-packages for common Linux distributions soon. Once those are offered,
-it will be the recommended way of installing Prometheus.
+Precompiled binaries for released versions are available in the
+[releases section](https://github.com/prometheus/prometheus/releases)
```
Comment (fabxc) [inline README.md]: `release section` or `"releases" section`?

**Comment by fabxc** [comment]
:+1:

File: README.md (hunk 1 above)
Comment (beorn7) [inline README.md]: Went for *releases* section.

**Comment by juliusv** [comment]
:+1:

**Pull request merged**